O365-iOS-Microsoft-Graph-Connect-Swift/AuthenticationClass.swift

```diff
@@ -97,7 +103,12 @@ class AuthenticationClass {
     }
 
     func disconnect() {
-        try? authenticationProvider.remove(authenticationProvider.users().first)
+        let accounts = try? authenticationProvider.allAccounts()
```

Comment:
It can be simplified to:
 ```
let accounts = try? authenticationProvider.allAccounts()
guard let account = accounts?.first else { return }

try? authenticationProvider.remove(account)
```

Reply:
:+1: fixed